platform/representation/src/gestures/DropGesture.js

```diff
@@ -90,25 +107,61 @@ define(
                 }
             }
 
+            function createVirtualPanel(base, overlayId){
```

Comment:
> Perhaps as a new action that is triggered when a telemetry object is dropped on another telemetry object?

Ah, yes, that's exactly what I was thinking in the [previous comment](https://github.com/nasa/openmctweb/pull/301/files#r44797270)